requirements.txt

```diff
@@ -0,0 +1 @@
+httpx==0.13.3
```

Comment:
Do we need a requirements.txt if we have the dependencies inside our `setup.py`? Can we install dependencies through `pip install .` or something else?

Reply:
You do. Development requirements include testing tools and the like. I'm sure we'll expand on this. You generally don't run setup.py to run tests, etc. You install requirements locally, and run things on it before packaging it.

You can see the httpx library itself follows this pattern. Basically, although we support a wide variety of httpx versions, we pick one to test with.

Reply:
https://github.com/encode/httpx

Reply:
Basically, `setup.py` contains dependencies for users, and `requirements.txt` contains dependencies for library developers. `requirements.txt` is more strict.